pages/account/manage/profile.js

```diff
@@ -232,6 +236,14 @@ export default function Profile({ BASE_URL, profile, fileExists }) {
                       </p>
                     </div>
                   </div>
+                  <div className="mt-3">
+                    <Toggle
+                      text1="Enable?"
+                      text2="hide/show profile view and rank"
+                      enabled={isStatsPublic}
+                      setEnabled={setisStatsPublic}
```

Comment:
I think it should be...

```suggestion
                      setEnabled={setIsStatsPublic}
```